content/blog/vs-firefox-relay.md

```diff
@@ -109,7 +107,7 @@ Some noteworthy features:
 
 ### Conclusion
 
-Using email aliases isn't mainstream (yet) and has probably the same popularity as password managers 10 years ago. For this reason, we don't see Firefox Relay as a concurrent but rather a co-pilot in the fight to gain back control of our data.
+Using email aliases aren't mainstream (yet) and has probably the same popularity as password managers 10 years ago. For this reason, we don't see Firefox Relay as a concurrent but rather a co-pilot in the fight to gain back control of our data.
```

Comment:
I feel grammatically we should "is" instead of "are" here, I can't find an official reference about this though.

Reply:
Only reason I changed it is because `aliases ` is plural but it doesn't affect the overall article so I have no issue either way.